src/Components/CardboardListCallout/CardboardListCallout.tsx

```diff
@@ -88,18 +88,17 @@ const CardboardListCallout = <T extends unknown>({
                 {description && (
                     <Text className={styles.description}>{description}</Text>
                 )}
-                {listItems.length > 0 && (
```

Comment:
was there a case where we don't want search to show? 🤔 I was surprised it wasn't an option. But maybe we don't need it? I guess there is not much reason to have one, but in other places in the app we always show it even when the list is empty so this is good.

Reply:
no, we should have always showed the searchbox